package.json

```
@@ -11,6 +11,8 @@
   },
   "scripts": {
     "wp-env": "wp-env",
+    "lint:js": "wp-scripts lint-js",
+    "lint:js:fix": "npm run lint:js -- --fix",
```

Can we use syntax consistent with the other existing scripts (`-` instead of `:`)?

Also, the convention for formatting scripts is to use `format`, so I think this would be better named as `format-js`.
```suggestion
    "lint-js": "wp-scripts lint-js",
    "format-js": "npm run lint:js -- --fix",
```

---

Fixed.